charts/kube-job-notifier/Chart.yaml

```diff
@@ -24,5 +24,5 @@ version: 0.1.6
 appVersion: "v0.0.21"
```

Comment:
if you change something in the image **appVersion**, if you change something in the app: **version**